### ISSUE TITLE
Batchnorm from pretrained backbone but frozen i.e. (param.requires_grad = False)

### ISSUE THREAD

**Issue by schoudhary101** [pytorch/opacus#315]
Using a pretrained backbone like resnet50, has batchnorm, froze the backbone with (param.requires_grad = False) 
Added new layers after this with Group Norm. 
Unable to train. Any way out?

**Comment by Anonymani** [comment]
Hi,
Thanks for reaching out, could you please provide more details?
What errors are you getting?
May be a minimal code to repro?

**Comment by schoudhary101** [comment]
I am trying to train a CNN model which has a pretrained backbone like resnet50 with BN. I want to train a CNN model on a custom task but the layers after the backbone have GroupNorm Residual blocks(No BN). The backbone is frozen this backprop is not required for those layers and only the new GN Res blocks with GN are to be updates with gradients. 
Now, Opacus throws the error asking for replacement of BN with GN etc. but gradients aren't being calculated for BN layers. 

If I train the same archtecture with backbone also having GN with random initialised weights and the same layers after it then I am able to train.

**Comment by karthikprasad** [comment]
Good catch! That's indeed a bug. Let me send a fix.
